packages/muelu/src/Rebalancing/MueLu_Zoltan2Interface_def.hpp

```diff
@@ -95,7 +96,17 @@ namespace MueLu {
   void Zoltan2Interface<Scalar, LocalOrdinal, GlobalOrdinal, Node>::DeclareInput(Level& currentLevel) const {
     Input(currentLevel, "A");
     Input(currentLevel, "number of partitions");
-    Input(currentLevel, "Coordinates");
+    const ParameterList& pL = GetParameterList();
+    // We do this dance, because we don't want "ParameterList" to be marked as used.
+    // Is there a better way?
+    Teuchos::ParameterEntry entry = pL.getEntry("ParameterList");
```

Comment:
@aprokop Following up on our earlier conversation: Using `"ParameterList"` here is actually the correct thing to do, as that's the name that is accessed down below. The translation from `"repartition: params"` to `"ParameterList"` happens in the ParameterListInterpreter.